From dc6d2c546b938b106f4f8ba708989193d9a6c90d Mon Sep 17 00:00:00 2001
From: Artur Paszkiewicz <artur.paszkiewicz@intel.com>
Date: Fri, 29 Nov 2019 11:56:41 +0100
Subject: [PATCH] smp: fix the activity indicator states

The values for SOF and EOF were interchanged, which caused an incorrect
default value to be used (100b instead of 101b).

Signed-off-by: Artur Paszkiewicz <artur.paszkiewicz@intel.com>
---
 src/smp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/smp.c b/src/smp.c
index b893d76..2c089f8 100644
--- a/src/smp.c
+++ b/src/smp.c
@@ -61,8 +61,8 @@
 #define LED_ON	1
 #define LED_4HZ	2
 #define LED_I4HZ	3
-#define LED_SOF	4
-#define LED_EOF	5
+#define LED_EOF	4
+#define LED_SOF	5
 #define LED_2HZ	6
 #define LED_I2HZ	7
 
-- 
2.24.0

